rules.md

```diff
@@ -12,44 +12,46 @@ The game consists of a series of cards and tokens.
+The **Group Objective** is to **Research** and **Disseminate** 5 **Insights**. If the **Players** accomplish this the game ends and the **Players** win. 
 
-In addition to the main shared objective, **Players** also have an additional, personal objective to complete that is determined by their **Role** card. If the group objective is completed, Players who complete their personal objective Double Win. If the group objective is not completed all Players lose, even if they met their personal objective.
+In addition to the **Group Objective**, **Players** also have an additional, **Personal Objective** to complete that is determined by their **Role** card. If the **Group Objective** is completed, Players who complete their **Personal Objective** get a **Personal Victory** in addition to the **Group Victory**. If the group objective is not completed all Players **lose**, even if they met their personal objective.
+
+If the group **Run Out of Ideas** (i.e. a **Player** cannot draw a card from the **Ideas Deck** at the start of their turn) the game is lost.
 
 ## Action Types
 
-During the game, a **Player** on their current turn may perform up to 2 **Actions** from the list below.
+During the game, a **Player** on their current turn may perform up to 2 **Actions**. The list of possible **Action Types** are given below. **Players** can choose to play 2 of the same **Action Type**.
 
 - **Ponder**: Draw a card from the **Ideas Deck** add it to their **Hand**.
-- **Research**: Turns an **Idea** card in their **Hand** into an **Output** by placing it in front of them on the table.
-- **Disseminate**: **Player** places a **Open Access** or **Restricted Access** token onto one of their Research **Outputs**.
-- **Learn**: Select one of the Tools from your **Hand** or the three visable in the **Tool Shop** and place it under your **Role** Card such that you can see the effect. Each Player may only have 2 Tools active at any one time. If the Player already has two Tools active, on can be returned to their Hand to be **Relearned** at a later date.
+- **Research**: Turn an **Idea** card in their **Hand** into a **Research Output** by placing it in front of them on the table.
+- **Disseminate**: **Player** places a **Open Access** or **Restricted Access** token onto one of their **Research Outputs**.
+- **Learn**: Select a **Tool** from either your **Hand** or the **Tool Shop**, and place it under your **Role** Card such that you can see the effect. Each **Player** may only have 2 **Tools** active at any one time. If a **Player** with two active **Tools** chooses to **Learn** a new one, they must return one of the other active **Tools** to their **Hand**. It can then be **Relearned** at a later date.
+- **Teach**: Select a **Tool** from either your **Hand** or your **Active Tools** to the **Active Tools** of another **Player**. 
```

Comment:
Missing some words.

Reply:
Where exactly? Sorry not seeing it!

Reply:
It's the Teach line, missing the word 'give' or 'gift'

Reply:
ah yes, thanks!